config/environments/production.rb

```diff
@@ -45,8 +45,9 @@
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
-  config.ssl_options = { redirect: false, hsts: false }
+  disable_ssl = ENV['DISABLE_SSL'] == "true"
+  config.force_ssl = !disable_ssl
+  config.ssl_options = { redirect: disable_ssl, hsts: disable_ssl }
```

Comment:
This reads like we are sending HTTPS redirects and HSTS headers only when DISABLE_SSL=true, which is almost the opposite of what you'd expect. It's not really a problem since ssl_options is ignored when force_ssl=false, but it confused me for a minute when I went to set this up. Especially combined with the fact that `force_ssl = !(ENV['DISABLE_SSL'] == "true")` is almost a double negative but not quite.

It works though and I don't think it needs to be "fixed". Just giving some late feedback on readability.